result.go

```diff
@@ -72,3 +72,25 @@ func (r solution) Attempts() int {
 func (r solution) InputHash() []byte {
 	return r.hd
 }
+
+func stripVendor(path string, info os.FileInfo, err error) error {
+	if info.Name() == "vendor" {
+		if _, err := os.Lstat(path); err == nil {
+			if info.IsDir() {
+				return removeAll(path)
+			}
+
+			if (info.Mode() & os.ModeSymlink) != 0 {
```

Comment:
Ah, gotcha. I'll try to figure out how to create a junction dir and write a test that exercises this.

For what it's worth, if this erases the target of symlinks, then we're already doing that today.